callback_plugins/homsaplog.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import (absolute_import, division, print_function)
+from ansible.plugins.callback.default import CallbackModule as CallbackModule_default
+from ansible.plugins.callback import CallbackBase
+try:
+    # Ansible 2.3
+    from ansible.vars import strip_internal_keys
+except ImportError:
+    try:
+        # Anisble2.4
+        from ansible.vars.manager import strip_internal_keys
+    except ImportError:
+        # Ansible 2.5
+        from ansible.vars.clean import strip_internal_keys
+try:
+    import simplejson as json
+except ImportError:
+    import json
+import sys
+reload(sys).setdefaultencoding('utf-8')
+
+'''
```

Comment:
Deze mag dus eigenlijk bovenaan: "Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants."